content/1.7/back-office/catalog/discount/add-edit-cart-price-rule.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Adding and Editing cart rules
+weight: 1
+---
+
+# Cart Rule > Edit
+When you enter the Cart rule editing UI, there are 3 separate tabs - Information, Conditions and Actions. Each of them are combined together with the general cart rules functionality.
+
+The explanations of the input fields, dropdowns, switch toggle buttons and text fields are stated below:<br>
+
+## Information tab
+
+![Cart Rules: Information tab](/static/img/cart-rules-information-tab.png)
+
+- **Name** - it is a mandatory field, it can be set as unique rule name. There is a widget at the right of this field with multiple languages. The different languages can be set to the rule names for this certain Information tab form.
+- **Description** - it can be described where is the cart rule used for in more details.
+- **Code** - unique cart rule code can be set here, or simply by clicking Generate and Prestashop will generate the rule automatically with unique code name. It will become as voucher from front end side.
```

Comment:
```suggestion
- **Code** - text field - 
If the user decide to use a Code for the cart rule then the code has to be unique to validate the form otherwise a error is thrown: "This cart rule code is already used (conflict with cart rule %Id of the cart rule%).
By clicking Generate and Prestashop will generate the rule automatically with unique code name. It will become as voucher.
If the code is left empty then the cart rule will automatically be applied to customers matching the condition.
```